config.json

```diff
@@ -264,75 +264,147 @@
       {
         "groupId": "androidx.compose.animation",
         "artifactId": "animation",
-        "version": "1.4.3",
-        "nugetVersion": "1.4.3.2",
+        "version": "1.5.0",
+        "nugetVersion": "1.5.0",
         "nugetId": "Xamarin.AndroidX.Compose.Animation",
         "dependencyOnly": false
       },
+      {
+        "groupId": "androidx.compose.animation",
+        "artifactId": "animation-android",
+        "version": "1.5.0",
+        "nugetVersion": "1.5.0",
+        "nugetId": "Xamarin.AndroidX.Compose.AnimationAndroid",
```

Comment:
All of our existing packages treat the `artifactId` hyphens as full separators and use periods as separators for the `nugetId`.  For example:
```
        "groupId": "androidx.compose.material",
        "artifactId": "material-icons-core",
        "nugetId": "Xamarin.AndroidX.Compose.Material.Icons.Core",
```

I think this should be
```
Xamarin.AndroidX.Compose.Animation.Android
```

Same for all other new packages in this PR (looks like around 20 of them).

Reply:
Fixed.